### ISSUE TITLE
Add an options setting to override logo image (ie. a product image)

### ISSUE THREAD

**Issue by siebird** [enupal/stripe#156]
Currently, the logo is passed to the new Stripe Checkout as the product image for single payments. Ideally, I'd like to dynamically pass an image URL to [override that setting](https://d.pr/i/oSjsXy).

**Comment by andrelopez** [comment]
Hi @siebird We just released Stripe Payments v3.3.0 adding support to this feature. 

```twig
{% set options = {
    checkoutImages: {0:"https://yourdomain.com/assets/docs/test.png"}
} %}

{{ craft.enupalstripe.paymentForm('handle', options) }}
```

Docs: https://docs.enupal.com/stripe-payments/templating/paymentform.html#options

**Comment by siebird** [comment]
@andrelopez thanks for adding this!
